GNUmakefile

```diff
@@ -17,6 +17,9 @@ EXTERNAL_TOOLS=\
 # list only our .go files i.e. exlcudes any .go files from the vendor directory
 GOFILES_NOVENDOR = $(shell find . -type f -name '*.go' -not -path "./vendor/*")
 
+VERSION=$(shell cat VERSION)
```

Comment:
Lets form the image prefix based on the branch name.
Psuedo-code:
If branch == "master"
  IMAGE-PREFIX = ""
else
  IMAGE-PREFIX = "`git branch`-"

Reply:
Use the above IMAGE-PREFIX while building and pushing docker images.
Eg. docker build -t openebs/maya:${IMAGE-PREFIX}-ci

Reply:
ok, will change

Reply:
This approach may have to face this: There are two branches in git: local branch and upstream branch. git branch gives local branch names(like ganesh-patch-fix). We are interested in upstream branch only - cstor-feature-integration. I tried with a couple of commands and found out that it is not easy(no direct cmds and no stable solution) to pick a particular upstream branch locally. Also, we specify the upstream branch from the PR initially from browser UI. @avishnu